libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL.h

```diff
@@ -103,6 +103,11 @@ class WiFiClientSecure : public WiFiClient {
     // Sets the requested buffer size for transmit and receive
     void setBufferSizes(int recv, int xmit);
 
+    // Returns whether MFLN negotiation for the above buffer sizes succeeded (after connection)
+    int getMFLNStatus() {
+      return connected() ? br_ssl_engine_get_mfln_negotiated(_eng) : false;
+    }
```

Comment:
why not 
```
    bool getMFLNStatus() {
      return connected() && br_ssl_engine_get_mfln_negotiated(_eng);
    }
```

Reply:
What @d-a-v said :stuck_out_tongue_closed_eyes:

Reply:
Maybe I get paid by the trinary?  I'll clean it up.